extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -120,6 +121,12 @@ export class AmpIframe extends AMP.BaseElement {
      * @private {?string}
      */
     this.targetOrigin_ = null;
+
+    /**
+     * @return {!../../../src/service/url-impl.Url}
+     * @private
+     */
+    this.getUrlService_ = once(() => Services.urlForDoc(this.element));
```

Comment:
This `once` stuff isn't necessary, neither access is in the fast path.

Reply:
Will remove in a follow-up PR.

Reply:
[Already have one](https://github.com/ampproject/amphtml/pull/15614). 😉